transaction_test.go

```diff
@@ -201,8 +201,8 @@ func TestSubmitTransaction_Invalid(t *testing.T) {
 		require.Error(t, result.Error)
 
 		assert.IsType(t, &types.FlowError{}, result.Error)
-		assert.IsType(t, &fvm.InvalidProposalKeySequenceNumberError{}, result.Error.(*types.FlowError).FlowError)
-		assert.Equal(t, invalidSequenceNumber, result.Error.(*types.FlowError).FlowError.(*fvm.InvalidProposalKeySequenceNumberError).ProvidedSeqNumber)
+		assert.IsType(t, &fvmerrors.InvalidProposalSeqNumberError{}, result.Error.(*types.FlowError).FlowError)
+		//assert.Equal(t, invalidSequenceNumber, result.Error.(*types.FlowError).FlowError.(*fvmerrors.InvalidProposalSeqNumberError).ProvidedSeqNumber)
```

Comment:
non-public field

Reply:
yeah, I'm going to expose them as getter methods on an upcoming pr

Reply:
@jordanschalm Can you please add a TODO with an explanation for why this is commented out?

Reply:
https://github.com/onflow/flow-emulator/pull/41/commits/598df7abce4c17e4a0beff7c566df9e3b7993857

Reply:
The pr to expose the field went into flow-go yesterday: https://github.com/onflow/flow-go/pull/618

And I got this fixed in a emulator [PR](https://github.com/onflow/flow-emulator/pull/43) here https://github.com/onflow/flow-emulator/pull/43/files#diff-fa3545436305a9d76b9af6bfe8d963d15c8f223bebfecf7a82f353eff5e792daR215